src/wirecraft/client/constants.py

```diff
@@ -8,3 +8,5 @@
 FPS = 30
 PADDING = 20
 FLAGS = pygame.FULLSCREEN | pygame.NOFRAME | pygame.SCALED
+LEVEL = 1
+DEBUG = False
```

Comment:
New debug options just dropped in the last version 🚀

Reply:
No debug option for the debug text atm :rocket:

Reply:
Make it yourself after merging main into your branche 🚀 🌕

Reply:
I am still seeing DEBUG here